wiki/wiki/doctype/wiki_page/wiki_page.py

```diff
@@ -283,6 +303,7 @@ def get_context(self, context):
 
 	def get_items(self, sidebar_items):
 		topmost = frappe.get_value("Wiki Group Item", {"wiki_page": self.name}, ["parent"])
+		wikiSpace = frappe.db.sql("""select name from `tabWiki Space`""")
```

Comment:
Like I said please don't use raw sql unless absolutely necessary. 
Also wouldn't this fetch all wiki spaces? Do we need it?

Also try to use snake case for variables. ie `wiki_space`.